src/System.ServiceProcess.ServiceController/src/System/ServiceProcess/ServiceController.cs

```diff
@@ -449,12 +449,6 @@ private static bool CheckMachineName(string value)
         }
 
         public void Close()
```

Comment:
This change is going to change the behavior with user-overiden Dispose methods. Implementing `Close` by calling `Dispose` is fine - not sure why you have changed it.

Reply:
To match NETFX. It calls Close() to reset state, without setting `disposed`, for example when names need to be regenerated. Once `disposed` is set, the class can no longer be used. Do you still want me to change it? I suppose I could just rename it Reset().

Reply:
Commented.